From 3d58062d67579b794decb4b77fb4640185d0a90b Mon Sep 17 00:00:00 2001
From: Leon Marz <main@lmarz.org>
Date: Fri, 17 Sep 2021 12:26:03 +0200
Subject: [PATCH 5/5] tmp audio offset

We're not using audaspace, which has not been tested by upstream
so some functions didn't have all parameters

Already fixed in upstream
---
 source/blender/blenkernel/intern/sound.c    | 3 ++-
 source/blender/sequencer/intern/strip_add.c | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/source/blender/blenkernel/intern/sound.c b/source/blender/blenkernel/intern/sound.c
index b4a1993..8737968 100644
--- a/source/blender/blenkernel/intern/sound.c
+++ b/source/blender/blenkernel/intern/sound.c
@@ -1312,7 +1312,8 @@ void BKE_sound_move_scene_sound(Scene *UNUSED(scene),
                                 void *UNUSED(handle),
                                 int UNUSED(startframe),
                                 int UNUSED(endframe),
-                                int UNUSED(frameskip))
+                                int UNUSED(frameskip),
+                                double UNUSED(audio_offset))
 {
 }
 void BKE_sound_move_scene_sound_defaults(Scene *UNUSED(scene), Sequence *UNUSED(sequence))
diff --git a/source/blender/sequencer/intern/strip_add.c b/source/blender/sequencer/intern/strip_add.c
index 0ca5968..3f4fddf 100644
--- a/source/blender/sequencer/intern/strip_add.c
+++ b/source/blender/sequencer/intern/strip_add.c
@@ -463,7 +463,8 @@ Sequence *SEQ_add_sound_strip(Main *bmain,
 Sequence *SEQ_add_sound_strip(Main *UNUSED(bmain),
                               Scene *UNUSED(scene),
                               ListBase *UNUSED(seqbase),
-                              SeqLoadData *UNUSED(load_data))
+                              SeqLoadData *UNUSED(load_data),
+                              const double UNUSED(audio_offset))
 {
   return NULL;
 }
-- 
2.33.0

